src/client/locale/en/translation.ts

```diff
@@ -2,8 +2,6 @@ const translationEn = {
   general: {
     emailDomain: 'gov.sg',
     shortUrlPrefix: 'go.gov.sg/',
-    // If you are modifying appTitle, then you will need to also modify the title tag content
-    // of index.html, 404.error.ejs, and 500.error.ejs.
```

Comment:
what was the context behind the removal of this?

Reply:
Thought about it and I have totally overlooked the importance of it. Will be reverting the change on this. Thanks for picking this up!